### PR TITLE
Updated autorest generated files to latest INT state. 

### PR DESCRIPTION
**Pull request by max-mironov**
Updated autorest generated files to latest INT state && Fixed build issues.

### CODE REVIEW

File: src/commands/analytics/log-flow.ts [hunk 1]
```diff
@@ -116,7 +116,7 @@ export default class ShowLogFlowCommand extends AppCommand {
 
     // adding properties
     if (logEntry.properties != null) {
-      const logProperties: { [propertyName: string]: string } = JSON.parse(logEntry.properties);
+      const logProperties: { [propertyName: string]: string } = JSON.parse(logEntry.properties.propertyName);
```
Comment (max-mironov) [inline src/commands/analytics/log-flow.ts]: Hey guys could you please take a look if this change is correct or I'm missing something?

Comment (christav) [reply src/commands/analytics/log-flow.ts]: @yimysty Could you take a look at this and verify the change?

Comment (YiMysty) [reply src/commands/analytics/log-flow.ts]: sure!

Comment (YiMysty) [reply src/commands/analytics/log-flow.ts]: image
the response is like this.. not quite get this part, seems there not `propertyName` property under `properties`?

Comment (christav) [reply src/commands/analytics/log-flow.ts]: Is that you saying the change is good, or that you'll look at it?

Comment (YiMysty) [reply src/commands/analytics/log-flow.ts]: I will look at it.

File: src/util/apis/generated/models/genericLog.js [hunk 2]
```diff
@@ -144,11 +164,25 @@ GenericLog.prototype.mapper = function () {
...
+          serializedName: 'message_id',
+          type: {
+            name: 'String'
+          }
+        },
         properties: {
           required: false,
           serializedName: 'properties',
           type: {
-            name: 'String'
```
Comment (YiMysty) [inline src/util/apis/generated/models/genericLog.js]: found the change, is that propertyName a fancy writing methods to indicate the key?

File: src/util/apis/generated/models/genericLog.js [hunk 3]
```diff
@@ -144,11 +164,25 @@ GenericLog.prototype.mapper = function () {
...
+            name: 'Dictionary',
+            value: {
+                required: false,
+                serializedName: 'StringElementType',
+                type: {
+                  name: 'String'
+                }
+            }
```
Comment (YiMysty) [inline src/util/apis/generated/models/genericLog.js]: To make the whole part display more easily...

**Comment by christav** [comment]
@max-mironov I just send your fork a PR for what I think is the correct log-flow change. If you can merge that this PR will update and then we can merge it. Thanks!

**Comment by max-mironov** [comment]
@christav Thanks for looking at and fixing! The PR has been merged.

**Pull request merged**